themes/digital.gov/src/scss/new/_navigation.scss

```diff
@@ -0,0 +1,19 @@
+@use "uswds-core" as *;
+
+.usa-nav__secondary-links {
+  .usa-nav__secondary-item {
+    @include at-media("desktop") {
+      padding-left: units(0.5);
+      & + .usa-nav__secondary-item::before {
+        background-color: color("base-lighter");
+        display: inline-block;
+        width: units("1px");
+        height: units(2);
+        content: "";
+        margin-right: units(0.5);
+        vertical-align: middle;
+        padding-right: 0;
+      }
+    }
+  }
+}
```

Comment:
This is the alternate solution proposed in the PR https://github.com/uswds/uswds/pull/4963#discussion_r1095960947.

This is fine since we'll update to the USWDS version with the fix once it's released.

Reply:
We can make these styles more readable by:
1. Simplifying selector
2. Adding indents
3. Sorting by alphabetical order

```scss
.usa-nav__secondary-links .usa-nav__secondary-item {
  @include at-media("desktop") {
    padding-left: units(0.5);
    
    & + .usa-nav__secondary-item::before {
      background-color: color("base-lighter");
      content: "";
      display: inline-block;
      height: units(2);
      margin-right: units(0.5);
      padding-right: 0;
      vertical-align: middle;
      width: units("1px");
    }
  }
}
```